plugout_pipewire.c

```diff
@@ -84,13 +84,11 @@ static long pipewire_open(enum plugout_endian *endian, long rate, long *buffer_b
 						    &pipewire_data);
 
 	// connect the stream
-	// TODO: do we need the realtime flag?
```

Comment:
I did not really know what I was doing when writing this code, but at least I was suspicious of this ;-)

Reply:
I don't know why the tutorials even use it 🤷🏻‍♀️